src/components/chips/js/chipsController.js

```diff
@@ -280,6 +280,11 @@ MdChipsCtrl.prototype.appendChip = function(newChip) {
   // Check for a null (but not undefined), or existing chip and cancel appending
   if (newChip == null || this.items.indexOf(newChip) + 1) return;
 
+	// Assign a value to the ngModel bound variable if it is not defined yet
+	if (!this.ngModelCtrl.$viewValue) {
+		this.ngModelCtrl.$setViewValue(this.items);
```

Comment:
@DevVersion I'm not quite sure that the `$render` function is the best place for setting the `$viewValue`: the `ngModel` would be directly set to `[]`while I think we should wait for a chip to be appended to set the value to `[…]`